pkg/graph/schema.resolvers_test.go

```diff
@@ -487,4 +488,274 @@ func (s GraphQLTestSuite) TestFetchSystemIntakeQuery() {
 	s.Equal(businessOwner, resp.SystemIntake.BusinessOwner)
 	s.Equal(businessOwnerComponent, resp.SystemIntake.BusinessOwnerComponent)
 	s.Nil(resp.SystemIntake.BusinessNeed)
+	s.Nil(resp.SystemIntake.BusinessCase)
+}
+
+func (s GraphQLTestSuite) TestFetchBusinessCaseForSystemIntakeQuery() {
+	ctx := context.Background()
+
+	intake, intakeErr := s.store.CreateSystemIntake(ctx, &models.SystemIntake{
+		Status:      models.SystemIntakeStatusINTAKESUBMITTED,
+		RequestType: models.SystemIntakeRequestTypeNEW,
+	})
+	s.NoError(intakeErr)
+
+	businessCase, businessCaseErr := s.store.CreateBusinessCase(ctx, &models.BusinessCase{
+		SystemIntakeID: intake.ID,
+		Status:         models.BusinessCaseStatusOPEN,
+		EUAUserID:      "TEST",
+	})
+	s.NoError(businessCaseErr)
+
+	var resp struct {
+		SystemIntake struct {
+			ID           string
+			BusinessCase struct {
+				ID                   string
+				AlternativeASolution struct {
+					Cons *string
+				}
+				LifecycleCostLines []struct {
+					Phase *string
+				}
+			}
+		}
+	}
+
+	// TODO we're supposed to be able to pass variables as additional arguments using client.Var()
+	// but it wasn't working for me.
+	s.client.MustPost(fmt.Sprintf(
+		`query {
+			systemIntake(id: "%s") {
+				id
+				businessCase { 
+					id
+					alternativeASolution {
+						cons
+					}
+					lifecycleCostLines {
+						phase
+					}
+				}
+			}
+		}`, intake.ID), &resp)
+
+	s.Equal(intake.ID.String(), resp.SystemIntake.ID)
+
+	respBusinessCase := resp.SystemIntake.BusinessCase
+	s.Equal(businessCase.ID.String(), respBusinessCase.ID)
+	s.Nil(respBusinessCase.AlternativeASolution.Cons)
```

Comment:
In the instance where there is no solution here, the response looks like:
```
alternativeASolution {
  cons: null
}
```

Is that what we would expect?

Reply:
shouldn't we just expect `null`? I'm not totally sure how I would expect that to resolve...

Reply:
@Igarfinkle that's what I would have expected, too, sooo... 🤷‍♀️